tests/tasks/test_s3_share_manager.py

```diff
@@ -1414,73 +1415,3 @@ def test_delete_dataset_bucket_key_policy_existing_policy_with_no_additional_tar
         # Then
         kms_put_key_policy_mock.assert_called()
         kms_put_key_policy_mock.assert_called_with(source_environment.AwsAccountId, 'eu-central-1', kms_get_key_mock.return_value, "default", json.dumps(remaining_policy))
-
-
-def test_handle_share_failure(
```

Comment:
minor: as these tests are removed, I guess there would be a "slight" coverage drop as the functions are still there with just logging

Reply:
The test was written to check that we send an email if there is a failure for shares. However, we stopped to create an alarm at some point (`AlarmService#trigger_revoke_table_sharing_failure_alarm`), so basically this test didn't test anything. The only thing we will lose is a coverage of writing logs, but I think it should be ok.